src/VisualStudio/TestUtilities/OutOfProcess/CSharpInteractiveWindow_OutOfProc.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.VisualStudio.Test.Utilities.InProcess;
+
+namespace Roslyn.VisualStudio.Test.Utilities.OutOfProcess
+{
+    public class CSharpInteractiveWindow_OutOfProc : InteractiveWindow_OutOfProc<CSharpInteractiveWindow_InProc>
```

Comment:
When I first took a look at this system, I was deeply confused by the separation of what ran in-proc vs. out-of-proc. The decorators helped me stay sane, I expect they'll help others as well. Thoughts?